app/cdap/components/CommonRecipeForm/index.tsx

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { FormControl, TextField, Typography, Box, Button } from '@material-ui/core';
+import styled from 'styled-components';
+import T from 'i18n-react';
+import { TextareaAutosize } from '@material-ui/core';
+import { ICommonRecipeFormProps, IRecipeData } from 'components/CommonRecipeForm/types';
+
+const FormFieldWrapper = styled(Box)`
+  width: calc(100% - 60px);
+  margin-right: 60px;
+  margin-bottom: 15px;
+  margin-top: 15px;
+`;
+
+const CommonLabelStyle = styled(Typography)`
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 150%;
+  letter-spacing: 0.15;
+`;
+
+const LabelErrorStyle = styled(CommonLabelStyle)`
```

Comment:
rename to ErrorLabel

we don't need style or component or array or object in variable names